services/sidecar/.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 bindings/I*
 bindings/L1Oracle.go
+*/**/forge-std
```

Comment:
why this is added?

Reply:
it is trying to include this in the repo, and probably it should not be there, right?